menustyle.css

```diff
@@ -0,0 +1,48 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Rock Salt', cursive;
+}
+header {
+    padding: 50px;
+}
+body {
+    background-image: url("trump.jpg");
+    background-size: cover;
```

Comment:
This is currently repeating the Trump image when window height is more than image height. Is this the desired behaviour? Doesn't seem to match up with the rest of the slick styling 😉